### ISSUE TITLE
🐛 [BUG] JIT Scripting xyz_to_angles

### ISSUE THREAD

**Issue by emilannevelink** [e3nn/e3nn#493]
**Describe the bug**
An error occurs when using `e3nn.o3._rotation.xyz_to_angles` in a scripted module.

The error is "Expected a value of type 'float' for argument 'p' but instead found type 'int'." This is a simple fix by changing the default `p` value from an int to a float ie `2` to `2.`.

Current code
`xyz = torch.nn.functional.normalize(xyz, p=2, dim=-1)  # forward 0's instead of nan for zero-radius`

Proposed code:
`xyz = torch.nn.functional.normalize(xyz, p=2., dim=-1)  # forward 0's instead of nan for zero-radius`

**To Reproduce**
```
import torch
from e3nn.o3._rotation import xyz_to_angles
from e3nn.util.jit import script

pos = torch.tensor([[0.,1.,1.]])
angles = xyz_to_angles(pos)
print(angles)

class XYZAngles(torch.nn.Module):
    def __init__(self):
        super().__init__()
    def forward(self, xyz):
        return xyz_to_angles(xyz)

mod = XYZAngles()
print(mod(pos))
scripted = script(mod)
print(scripted(pos))
```
**Expected behavior**

**Environment (please complete the following information):**
 - OS: ios
 - python version (`python --version`) 3.9.20
 - python environment (commands are given for python interpreter):
   - e3nn version (`import e3nn; e3nn.__version__`): 0.5.3
   - pytorch version (`import torch; torch.__version__`) 2.5.1
   - pytorch_geometric version (`import torch_geometric; torch_geometric.__version__`) 2.6.1
 - (if relevant) GPU support with CUDA
   -  cuda Version according to nvcc (`nvcc --version`)
   -  cuda version according to pyTorch (`import torch; torch.version.cuda`)

**Screenshots**
N/A

**Additional context**
N/a

**Comment by mitkotak** [comment]
Thanks for the bug report ! https://github.com/e3nn/e3nn/pull/494 should fix it. Also as an FYI we are starting to nudge folks towards the new `torch.compile` interface given that `torchscript` is planning to be deprecated.

```
import torch
from e3nn.o3._rotation import xyz_to_angles
import e3nn
from e3nn.util.jit import script
e3nn.set_optimization_defaults(jit_script_fx=False)

pos = torch.tensor([[0.,1.,1.]])
angles = xyz_to_angles(pos)
print(angles)

class XYZAngles(torch.nn.Module):
    def __init__(self):
        super().__init__()
    def forward(self, xyz):
        return xyz_to_angles(xyz)

mod = XYZAngles()
print(mod(pos))
compiled= torch.compile(mod, fullgraph=True)
print(compiled(pos))
scripted = script(mod)
print(scripted(pos))
```

**Comment by emilannevelink** [comment]
Thank you!
